core/server/worker/src/main/java/alluxio/worker/page/PagedBlockStore.java

```diff
@@ -188,20 +188,20 @@ private void commitBlockToMaster(PagedBlockMeta blockMeta) {
     try {
       bmc.commitBlock(mWorkerId.get(), mPageMetaStore.getStoreMeta().getUsedBytes(), DEFAULT_TIER,
           DEFAULT_MEDIUM, blockId, blockMeta.getBlockSize());
+      BlockStoreLocation blockLocation =
+              new BlockStoreLocation(DEFAULT_TIER, getDirIndexOfBlock(blockId));
+      for (BlockStoreEventListener listener : mBlockStoreEventListeners) {
+        synchronized (listener) {
+          listener.onCommitBlockToLocal(blockId, blockLocation);
+        }
+      }
```

Comment:
Move this to `commitBlock` where it is commited to the local page store after `pageStoreDir.commit` and `mPageMetaStore.commit`